gen_gold_lf.py

```diff
@@ -52,7 +52,7 @@ def process_cat(fpath, vmax_opath, field=None, rand_paths=[]):
     else:
         rand  = None
 
-    gama_vmax = vmaxer(gama_zmax, zmin, zmax, extra_cols=['MCOLOR_0P0', 'DDPMALL_0P0_VISZ'], rand=rand)
+    gama_vmax = vmaxer(gama_zmax, zmin, zmax, extra_cols=['MCOLOR_0P0', 'DDPMALL_0P0_VISZ', 'FIELD'], rand=rand)
```

Comment:
why's this necessary?

Reply:
This made it far easier to test the V/Vmax plots for each of the three fields based on the existing code.

Reply:
Ok, makes sense!